packages/ui/components/ui/tx/view/value.tsx

```diff
@@ -7,15 +7,16 @@ import { getDisplayDenomExponent } from '@penumbra-zone/types/src/denom-metadata
 
```

Comment:
```
import { getDisplayDenomExponent } from '@penumbra-zone/types';
```

instead of :

```
import { getDisplayDenomExponent } from '@penumbra-zone/types/src/denom-metadata';
```

(since you've added it as an export from the types package)